### PR TITLE
Fix aircraft idle speed and turn speed inconsistencies

### LINKED ISSUE #19278: Aircraft Idle Speed Unaffected by SpeedMultiplier or Pausing
The SpeedMultiplier trait has no effect on aircraft idle speed, which means it's possible for aircraft to move faster while idling than when flying if they're affected by a condition intended to reduce aircraft speed.

Turn speed is also not affected.

If you pause the Aircraft trait while the aircraft is idling, this also has no effect.

Came across this as I was attempting to make a weapon that freezes aircraft in place, but it only worked when the aircraft wasn't idling.

The FlyIdle activity class references the speeds directly from the Aircraft Info without applying any modifiers as in the Aircraft class itself.

Ideally turn speeds should be affected too, otherwise applying a zero speed multiplier causes an idling aircraft to just spin in place.

### PR DESCRIPTION
**Pull request by reaperrr**
Fixes #19278.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Air/Aircraft.cs [hunk 1]
```diff
@@ -268,6 +268,19 @@ public WAngle Roll
...
+		public WAngle GetTurnSpeed(bool isIdleTurn)
+		{
+			// A MovementSpeed of zero indicates either a speed modifier of zero percent or that the trait is paused or disabled.
+			// Bail early in that case.
+			if ((isIdleTurn && IdleMovementSpeed == 0) || MovementSpeed == 0)
+				return WAngle.Zero;
+
+			var turnSpeed = isIdleTurn ? IdleTurnSpeed ?? TurnSpeed : TurnSpeed;
+
+			turnSpeed = new WAngle(Util.ApplyPercentageModifiers(turnSpeed.Angle, speedModifiers).Clamp(1, 1024));
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/Air/Aircraft.cs]: `return new WAngle(...)`.

**Review by pchote — COMMENTED** [review]
Fixes confirmed, and LGTM overall.

Just two small nits:
* The compiler is smart enough that the "Aircraft turn speed getters perf optimization" commit shouldn't change anything in practice

**Comment by abcdefg30** [comment]
> The compiler is smart enough that the "Aircraft turn speed getters perf optimization" commit shouldn't change anything in practice

Fwiw I find it easier to read the new order (||).

**Comment by reaperrr** [comment]
Updated.

> Fwiw I find it easier to read the new order (||).

Same here, that was part of the reason I made this change. Updated commit title/description to reflect this.

**Review by pchote — APPROVED** [review]

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/e30e8ac389c0f637828690802c800715f679cb72

**Pull request merged**